src/cascadia/TerminalCore/TerminalSelection.cpp

```diff
@@ -517,11 +533,6 @@ void Terminal::SelectHyperlink(const SearchDirection dir)
     _ScrollToPoint(_selection->end);
 }
 
-bool Terminal::SelectionIsTargetingUrl() const noexcept
-{
-    return _selectionIsTargetingUrl;

```

Comment:
so `_selectionIsTargetingUrl` is now only used to know whether we are in URL jump mode?

Reply:
Ah, it's only so that we don't jump to the link we just jumped to when moving forward?

Reply:
Yup. It's so that we skip over the one that's currently selected.